yarn/Dockerfile

```diff
@@ -0,0 +1,16 @@
+# Use the base App Engine Docker image, based on debian jessie.
+FROM gcr.io/google-appengine/debian8
+
+# Install updates and dependencies
+RUN apt-get update -y && apt-get install --no-install-recommends -y -q curl python build-essential git ca-certificates libkrb5-dev imagemagick && \
+    apt-get clean && rm /var/lib/apt/lists/*_*
+
+# Install the latest LTS release of nodejs
+RUN mkdir /nodejs && curl https://nodejs.org/dist/v6.9.1/node-v6.9.1-linux-x64.tar.gz | tar xvzf - -C /nodejs --strip-components=1
+ENV PATH $PATH:/nodejs/bin
```

Comment:
The first part of this `Dockerfile` copies a portion of the `Dockerfile` for `npm`.  When a new Node.js vesion will be used, the version will need to be updated in the `npm` and `yarn` Dockerfiles separately.  For this repo, I'm not sure if this is an issue.  @Philmod What do you think?